Tests/SonarQube.TeamBuild.Integration.Tests/TfsLegacyCoverageReportProcessorTests.cs

```diff
@@ -79,12 +79,14 @@ public void ReportProcessor_NoUrlsFound()
             TfsLegacyCoverageReportProcessor processor = new TfsLegacyCoverageReportProcessor(urlProvider, downloader, converter);
 
             // Act
-            bool result = processor.ProcessCoverageReports(context, settings, logger);
+            bool initResult = processor.Initialise(context, settings, logger);
+            bool result = processor.ProcessCoverageReports();
 
             // Assert
             urlProvider.AssertGetUrlsCalled();
             downloader.AssertDownloadNotCalled(); // no urls returned, so should go any further
             converter.AssertConvertNotCalled();
+            Assert.IsTrue(initResult, "Expecting true: processor was initialised");
```

Comment:
This assertion should be after line 82; no point in trying to call "ProcessCoverageReports" if initialisation failed.